tsconfig.json

```diff
@@ -1,14 +1,14 @@
 {
   "references": [
-    { "path": "packages/core" },
-    { "path": "packages/environment-ember-loose" },
-    { "path": "packages/environment-ember-template-imports" },
-    { "path": "packages/environment-glimmerx" },
-    { "path": "packages/template" },
-    { "path": "packages/type-test" },
-    { "path": "packages/vscode" },
-    { "path": "packages/scripts" },
-    { "path": "test-packages/test-utils" }
+    { "path": "./packages/core" },
```

Comment:
in one of the commits, it's explained that this change was needed so that you can ctrl+click to go to each of these tsconfigs.

It's kinda silly that tsc works with both path-formats but vscode only works with leading ./